Sources/Puree/Logger.swift

```diff
@@ -74,18 +73,30 @@ public final class Logger {
         return logs
     }
 
+    private func start() {
+        dispatchQueue.sync {
+            outputs.forEach { $0.start() }
+        }
+    }
+
     public func suspend() {
-        outputs.forEach { $0.suspend() }
+        dispatchQueue.sync {
+            outputs.forEach { $0.suspend() }
```

Comment:
Is this needed to execute on DispatchQueue?

Reply:
outputs must be treated in the same thread (threadsafe).